jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/AbstractConnectionPool.java

```diff
@@ -449,45 +441,6 @@ protected void onCreated(Connection connection)
     {
     }
 
-    /**
-     * @param connection the {@link Connection} that become idle
-     * @param close whether this pool is closing
-     * @return {@code true} to indicate that the connection is idle, {@code false} otherwise
-     * @deprecated Racy API. Do not use. There is no replacement.
-     */
-    @Deprecated(since = "12.0.8", forRemoval = true)
-    protected boolean idle(Connection connection, boolean close)
```

Comment:
Following your past arguments about deprecations, this shouldn't be deleted until version 12.2.x, why the change in opinion?